### PR TITLE
Add Omnibus definition for aerospike

### PR DESCRIPTION
**Pull request by ofek**
### Motivation

The binary wheels on PyPI are not yet compatible with OpenSSL 1.1.0+

### CODE REVIEW

File: omnibus/config/software/datadog-agent-integrations-py2.rb [hunk 1]
```diff
@@ -28,6 +28,10 @@
   dependency 'nfsiostat'
   # need kerberos for hdfs
   dependency 'libkrb5'
+
+  unless suse || arm?
```
Comment (KSerrania) [inline omnibus/config/software/datadog-agent-integrations-py2.rb]:
```suggestion
  unless suse? || arm?
```

File: omnibus/config/software/datadog-agent-integrations-py3.rb [hunk 2]
```diff
@@ -28,6 +28,10 @@
   dependency 'nfsiostat'
   # need kerberos for hdfs
   dependency 'libkrb5'
+
+  unless suse || arm?
```
Comment (KSerrania) [inline omnibus/config/software/datadog-agent-integrations-py3.rb]:
```suggestion
  unless suse? || arm?
```

**Review by albertvaka — APPROVED** [review]
Thanks for working on this guys :)

Are we okay with not having aerospike on suse, though?

**Comment by KSerrania** [comment]
@albertvaka it was already like this before, the definition was temporarily changed in #4954 to blacklist it on all platforms instead of SUSE and ARM only.

**Review by FlorianVeaux — APPROVED** [review]

**Pull request merged**